docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,87 @@
+# Release 353 (XXXXXXXX March 2021)
+
+## General
+
+* Fix aliasing of columns in common table expressions. ({issue}`6839`)
+* Fix potential incorrect query results for queries containing multiple `<` predicates. ({issue}`6896`)
+* Always show `SECURITY` clause in `SHOW CREATE VIEW`. ({issue}`6720`)
+* Fix reporting column references for aliased tables. ({issue}`6972`)
+* Fail `SHOW COLUMNS` when there are issues with getting information about table columns.
+  Previously empty list of columns was returned. ({issue}`6958`)
+* Improve parallelism of bucketed tables inserts. Inserts into bucketed tables can now be parallelized
+  within task using `task.writer-count` feature config. ({issue}`6924`)
+* Extend support for correlated subqueries including `UNNEST`. ({issue}`6326`, {issue}`6925`, {issue}`6951`)
+
+## Server RPM
+
+* Allow configuring process environment variables through `/etc/trino/env.sh`. ({issue}`6635`)
+
+## BigQuery connector
+
+* Allow for case-insensitive identifiers matching via `bigquery.case-insensitive-name-matching` config property. ({issue}`6748`)
+* Add support for `CREATE TABLE` and `DROP TABLE` statements. ({issue}`3767`)
+
+## Hive connector
+
+* Fix a failure when `INSERT` writes to a partition created by an earlier `INSERT` statement. ({issue}`6853`)
+* Correctly recognize directory objects created by AWS S3 web console. ({issue}`6992`)
+* Add support for `current_user()` in Hive defined views. ({issue}`6720`)
+* Improve insert parallelism when inserting into partitioned and bucketed tables. ({issue}`6866`)
+* Add support for reading and writing column statistics from Glue metastore. ({issue}`6178`)
+
+## Iceberg connector
+
+* Correctly recognize directory objects created by AWS S3 web console. ({issue}`6992`)
+
+## Kafka connector
+
+* Fix `Classloader` related issue seen in Worker for Schema Registry tables. ({issue}`6902`)
```

Comment:
Can we describe this in a way understood by users not familiar with class loaders?
cc @Praveen2112